src/sentry/static/sentry/app/utils/eventWaiter.tsx

```diff
@@ -61,10 +62,31 @@ class EventWaiter extends React.Component<Props, State> {
 
   pollHandler = async () => {
     const {api, organization, project, onIssueRecieved} = this.props;
-
-    const {firstEvent} = await api.requestPromise(
-      `/projects/${organization.slug}/${project.slug}/`
-    );
+    let firstEvent = null;
+
+    try {
+      ({firstEvent} = await api.requestPromise(
+        `/projects/${organization.slug}/${project.slug}/`
+      ));
```

Comment:
Kinda ugly with the parens. I feel like I'd just opt for 

```tsx
const resp = await api.requestPromise(
  `/projects/${organization.slug}/${project.slug}/`
);
firstEvent = resp.firstEvent;
```

Reply:
@EvanPurkhiser updated https://github.com/getsentry/sentry/pull/17994/commits/90fc2e3d360b53b0f7d7b30e738ea0aa36cdd1a9